_episodes/11-obsolete-gitlab.md

```diff
@@ -0,0 +1,94 @@
+---
+title: OBSOLETE - Introduction to GitLab
+teaching: 30
+exercises: 0
+questions:
+    - "What is the purpose of GitLab?"
+    - "How can the Field Team use GitLab?"
+keypoints:
+    - "GitLab helps organize and archive tasks"
+    - "There are several ways GitLab can be used by the Field Team"
+---
+
+## OTN's GitLab
+
+At OTN, GitLab is used for internal record keeping and collaborative code development. The main purpose (for non-computer scientists at OTN) is the ability to use GitLab Issues containing templates of task-lists to ensure the DAQ team NEVER forgets a step in data loading, and that no file is ever lost/forgotten in an inbox. We track all communications with researchers and archive all important information for future use.
+
+GitLab allows us to be accountable, thorough, and comprehensive in our data-loading process. This is essential for smooth communication and acts as a searchable archive of OTN's institutional memory.
+
+## Field Ops Gitlab
+
+As the Field team has grown, job descriptions have expanded, and remote work has become more common it is now important to start using some of the lessons learned by the Data team to improve Field Ops communications. For this reason, there is now a **Field Team GitLab** project - [https://gitlab.oceantrack.org/otnfield/OTN_Field_Ops](https://gitlab.oceantrack.org/otnfield/OTN_Field_Ops).
+
+Each OTN Field team member should have a GitLab account created - please fill out this [signup form for an account on GitLab](https://gitlab.oceantrack.org/users/sign_up) if you don't have an account already.
+
+This Field Ops Gitlab has been created as a place to track complex / important / outstanding issues so that they can be contributed to collaboratively and archived for future reference. This will be used in addition to Slack to make sure all Field Team members are communicating and information does not slip between the cracks of our busy schedules.
+
+Gitlab Issues should be created to track:
+
+- lost equipment
+- equipment returns
+- found equipment
+- RMAs, including the outcome as reported by the manufacturer, and all data files (these usually are just emailed)
+- deployment or metadata issues / questions from PIs that need digging into
+- outstanding purchase orders
+- anything that is too complicated to track via email and/or involves too many people
+
+Current members of this Gitlab Project include: all Field staff, members of the Project Management Office, some members of the Data Team. Please tag them in an issue (using the @ symbol) to prompt them for input.
+
+Field team members leave (for vacation, or permanently) and so a shared record of important events will be essential to ensure knowledge-transfer. Having the information in your email is a great first step, but it may become lost or buried and is not easily shared/searched by colleagues.
+
+![FieldOps GitLab Home Page](../fig/fieldops_gitlab_home.PNG)
+
+## How to Use GitLab
+
+Menu on the left-hand sidebar:
```

Comment:
mix of ending with punctuation vs no punctuation (not sure if we care about this)